internal/subshell/bash/bash.go

```diff
@@ -22,6 +22,9 @@ var rcFileName = ".bashrc"
 
 func init() {
 	escaper = osutils.NewBashEscaper()
+	if runtime.GOOS == "darwin" {
+		rcFileName = ".bash_profile"
```

Comment:
Nit: maybe comment why .bash_profile is being used instead of .bashrc on macOS.

Reply:
Good point!